iosApp/iosApp/Modules/ImprovementProposals/ImprovementProposalsWireframe.swift

```diff
@@ -67,15 +55,11 @@ extension DefaultImprovementProposalsWireframe {
         )
         let vc: ImprovementProposalsViewController = UIStoryboard(.improvementProposals).instantiate()
         let presenter = DefaultImprovementProposalsPresenter(
-            view: vc,
+            view: WeakRef(referred: vc),
```

Comment:
did you get a chance to test the deinit methods for the new .kt classes in the module to be deallocated properly?